README.md

```diff
@@ -13,10 +13,19 @@ or
 
 `conda install xarray-enmap`
 
+> ⚠️ If you want to export Zarr archives using the included command-line tool
+> `convert-enmap`, also install the packages `zarr` and `numcodecs`.
```

Comment:
I see you updated the environment.yml accordingly, but you could mention here that this requires you to use zarr<3 and, in consequence, python < 3.14